src/types.ts

```diff
@@ -1252,6 +1252,8 @@ export type ConnectionChangeEvent = {
 
 export type Event<StreamChatGenerics extends ExtendableGenerics = DefaultGenerics> = StreamChatGenerics['eventType'] & {
   type: EventTypes;
+  ai_message?: boolean;
```

Comment:
This should be String right?
```suggestion
  ai_message?: string;
```